package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-loadable",
-  "version": "5.5.0",
+  "version": "5.6.0",
```

Comment:
@MiguelCastillo ran into some issues with vite not rebuilding this module.  We think it may be because the version in the package.json didn't get bumped, so I'm bumping it before landing this PR.